### ISSUE TITLE
[16.0] web_chatter_position: Button to switch between sided and bottom views

### ISSUE THREAD

**Issue by mschulz-asphericon** [OCA/web#2724]
Previously in 15.0, next to the save/edit button there was an arrow button to switch between sided and bottom view , depending on how it's currently needed. 
Somehow this wasn't migrated. Can this please be added again?

![grafik](https://github.com/OCA/web/assets/88142010/64c51c8e-21d8-45e0-9b25-7ce84cca0f5a)

**Comment by rvalyi** [comment]
It's now handled with the web_chatter_position new module (same OCA/web repo) in version 16. I'm not sure about the why but I guess this answers your need.

**Comment by mschulz-asphericon** [comment]
@rvalyi Yes it's regarding the web_chatter_position , but the button of the screenshot is missing in v16, it seems to be handled only responsively or by the user setting. Or is there something I'm missing?

**Comment by chrisandrewmann** [comment]
Any news on this? I hadn't realised that the button to toggle chatter position originally existed but was removed since V16.
It would be really useful if reintroduced to V16, V17 onwards.

**Comment by trisdoan** [comment]
Hello @mschulz-asphericon, @chrisandrewmann , I re-introduced the button in https://github.com/OCA/web/pull/3040
